app/src/main/java/com/douglasbruce/tasky/features/event/navigation/EventNavigation.kt

```diff
@@ -20,27 +20,32 @@ internal const val eventFromDateArg = "eventFromDate"
 @VisibleForTesting
 internal const val eventToDateArg = "eventToDate"
 
+@VisibleForTesting
+internal const val eventIsEditingArg = "eventIsEditing"
+
 internal class EventArgs(
     val eventId: String?,
     val eventToDateMilli: Long,
-    val eventFromDateMilli: Long
+    val eventFromDateMilli: Long,
+    val eventIsEditing: Boolean,
 ) {
     constructor(savedStateHandle: SavedStateHandle) :
             this(
                 savedStateHandle[eventIdArg],
                 checkNotNull(savedStateHandle[eventToDateArg]),
-                checkNotNull(savedStateHandle[eventFromDateArg])
+                checkNotNull(savedStateHandle[eventFromDateArg]),
+                checkNotNull(savedStateHandle[eventIsEditingArg])
             )
 }
 
 fun NavController.navigateToNewEvent(dateMilli: Long) {
-    this.navigate("$eventNavigationRoute/$dateMilli/$dateMilli") {
+    this.navigate("$eventNavigationRoute/$dateMilli/$dateMilli/true") {
         launchSingleTop = true
     }
 }
 
-fun NavController.navigateToEvent(eventId: String) {
-    this.navigate("$eventNavigationRoute/$eventId") {
+fun NavController.navigateToEvent(toDateMilli: Long, fromDateMilli: Long, eventId: String, isEditing: Boolean) {
+    this.navigate("$eventNavigationRoute/$toDateMilli/$fromDateMilli/$isEditing?$eventIdArg=$eventId") {
```

Comment:
Why is `toDateMilli` and `fromDateMilli` needed as a nav argument here? I think just the date should be required, right?